src/en/administration/install_docker.md

```diff
@@ -16,39 +16,31 @@ mkdir -p volumes/pictrs
 sudo chown -R 991:991 volumes/pictrs
 ```
 
-Open up your `docker-compose.yml`, and make sure `LEMMY_EXTERNAL_HOST` for `lemmy-ui` is set to your correct host.
-
-```
-- LEMMY_INTERNAL_HOST=lemmy:8536
-- LEMMY_EXTERNAL_HOST=your-domain.com
-- LEMMY_HTTPS=false
-```
-
 If you'd like a different database password, you should also change it in the `docker-compose.yml` **before** your first run.
 
 After this, have a look at the [config file](configuration.md) named `lemmy.hjson`, and adjust it, in particular the hostname, and possibly the db password. Then run:
 
 `docker-compose up -d`
 
-You can access the lemmy-ui at `http://localhost:1235`
+You can access the lemmy-ui at `http://localhost:80`
 
-To make Lemmy available outside the server, you need to setup a reverse proxy, like Nginx. [A sample nginx config](https://github.com/LemmyNet/lemmy-ansible/blob/main/templates/nginx.conf), could be setup with:
+To make Lemmy available outside the server, you need to setup a reverse proxy, like Nginx. You can use the following simple proxy:
 
-```bash
-wget https://raw.githubusercontent.com/LemmyNet/lemmy-ansible/main/templates/nginx.conf
-# Replace the {{ vars }}
-# The default lemmy_port is 8536
-# The default lemmy_ui_port is 1235
-sudo mv nginx.conf /etc/nginx/sites-enabled/lemmy.conf
+```
+server_name your_domain;
+location / {
+    proxy_pass http://localhost:LEMMY_PORT;
+    include proxy_params;
+}
```

Comment:
I dont think this is a valid nginx config, because its missing the server block and listen port. I would also include parts for both http and https, as well as the necessary lines to get tls working (ssl_certificate etc).

 For Lets Encrypt it would also be good to include more detailed instructions, with a list of commands that you need to run (install letsencrypt package from ubuntu repo, request cert, edit nginx config, restart nginx).

Reply:
Letsencrypt adds those blocks for you in your nginx conf file automatically, but I'll add a guide for it there, as well as the whole server block.